### ISSUE TITLE
FR: Show only user apps that are not backed up

### ISSUE THREAD

**Issue by gerroon** [jensstein/oandbackup#94]
Hi

I can show only the user apps or the apps that are not backed up which is great. The thing is that the apps not backed up shows also the whole list of other system and core aps etc. So it seems to me that there is no way to show the apps that are not backup but owned by the user.

thanks

**Comment by jensstein** [comment]
yes, i am aware of this problem. sorting and filtering in generel needs a revision. 
thanks for reporting :)

**Comment by gerroon** [comment]
That would be very helpful.  Thanks for the hard work.

**Comment by ildar** [comment]
Then we'll need the "Ignore" feature for apps that inherently do not need backed up, namely cloud apps, e.g. GMail.
